app/config/config.yml

```diff
@@ -78,6 +78,9 @@ fos_user:
     db_driver: orm # other valid values are 'mongodb', 'couchdb' and 'propel'
     firewall_name: main
     user_class: Digip\RenovationBundle\Entity\User
+    from_email:
```

Comment:
These values should be in parameters.yml since this is not generic for this open source instance